tests/integration/states/test_pkg.py

```diff
@@ -506,6 +511,9 @@ def test_pkg_010_latest_with_epoch_and_info_installed(self):
         if not pkgmgr_avail(self.run_function, self.run_function('grains.items')):
             self.skipTest('Package manager is not available')
 
+        if (grains.get('os') == 'Amazon' and grains.get('osmajorrelease') != 2):
```

Comment:
would it be possible to change the pkg name to something that is available on amazon linux 1, instead of skipping the test so we ensure we are also testing this on amazon linux 1? I believe it just needs to be a package that includes an epoch

Reply:
I shall take a look at something else that we can install but typically would not be installed by default

Reply:
Using bash-doc instead of unavailable bash-completion